README.md

```diff
@@ -185,30 +189,105 @@ guarantees.
 and milestones are estimates, and subject to change.
 
+The following tables describe the artifacts published by this project. To take a dependency, follow
+the instructions in [Published Released](#published-releases) to include the BOM, and specify the
+dependency as follows, replacing `{{artifact-id}}` with the value from the "Artifact ID" column:
+
+```xml
+<dependency>
+  <groupId>io.opentelemetry</groupId>
+  <artifactId>{{artifact-id}}</artifactId>
+</dependency>
+```
+
+```groovy
+  implementation('io.opentelemetry:{{artifact-id}}')
+```
+
+### Bill of Material
+
+| Component                                    | Artifact ID               | Version                                                     |
+|----------------------------------------------|---------------------------|-------------------------------------------------------------|
+| [Bill of Materials (BOM)](./bom)             | `opentelemetry-bom`       | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Alpha Bill of Materials (BOM)](./bom-alpha) | `opentelemetry-bom-alpha` | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+
+### API
+
+| Component                         | Artifact ID             | Version                                                     |
+|-----------------------------------|-------------------------|-------------------------------------------------------------|
+| [API](./api/all)                  | `opentelemetry-api`     | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Context API](./context)          | `opentelemetry-context` | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Semantic Conventions](./semconv) | `opentelemetry-semconv` | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+
+### SDK
+
+| Component                              | Artifact ID                      | Version                                                     |
+|----------------------------------------|----------------------------------|-------------------------------------------------------------|
+| [SDK](./sdk/all)                       | `opentelemetry-sdk`              | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [SDK Common](./sdk/common)             | `opentelemetry-sdk-common`       | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Metrics SDK](./sdk/metrics)           | `opentelemetry-sdk-metrics`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Trace SDK](./sdk/trace)               | `opentelemetry-sdk-trace`        | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
+| [Log SDK](./sdk/logs)                  | `opentelemetry-sdk-logs`         | <!--VERSION_UNSTABLE-->1.14.0-alpha<!--/VERSION_UNSTABLE--> |
+| [SDK Testing](./sdk/testing)           | `opentelemetry-sdk-testing`      | <!--VERSION_STABLE-->1.14.0<!--/VERSION_STABLE-->           |
```

Comment:
Just from the name, I have no idea what "SDK Testing" is.

Reply:
Initially was trying to keep a description of the artifacts out of scope, but I think its useful so I've added an additional description column